1.10/tutorials/dcos-101/service-discovery.md

```diff
@@ -31,9 +45,14 @@ In this section, you will learn about DC/OS service discovery by exploring the d
   redis.marathon.mesos. 60  IN  A 10.0.0.43
   ```
 
-  The response tells us that the host for the `redis.marathon.mesos` is 10.0.0.43.
+  The response tells us that the host for the `redis.marathon.mesos` service is 10.0.0.43.
```

Comment:
the response tells us that there is an instance of the `red.... ` service running on....

not 100% happy with my wording, but `the host` seems wrong as there could be multiple instances.